src/components/KBadge/KBadge.vue

```diff
@@ -8,7 +8,7 @@
       'clickable': isClickable
     } ]"
     :style="badgeCustomStyles"
-    :tabindex="hidden ? -1 : 0"
+    :tabindex="hidden ? -1 : isClickable ? 0 : undefined"
```

Comment:
**Comment**: In the case of `isClickable` being true, the `.k-badge` element should probably just be a button.